docs/develop/bundled-packages.md

```diff
@@ -1,18 +1,13 @@
 # Bundled packages
 
-There are times when model code depends on internal packages that are not
-available on pypi. This code can be bundled as submodule under the model module
-but that may not work all the time. e.g. the in-memory model may refer to these
-packages as top level modules and that may not work when they become submodules.
-Bundled packages can help in that scenario.
+If your model depends on internal packages that are not available on PyPi, you can bundle this code with the Truss in the `packages` folder. These packages then become available as top level modules to the model class's code when it executes, whether it is running locally, in any Docker container, or on Baseten.
 
-Top level packages can be bundled with the truss by placing them in the
-`packages` folder under the truss directory. These packages then become
-available as top level modules to the model class's code when it executes in the
-local, docker or baseten deployed environments.
+{% hint style="info" %}
+When you import your Truss, the import mechanism adds everything in the Truss' root directory and packages directory to the path.
```

Comment:
May be we can just say that `import mechanism adds these packages to the the path`, the part about adding to root directory is probably a detail we can omit.